docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -1,37 +1,33 @@
 ## How to use it
 
-Since the v3 of Otter, you should use the component-extraction builder directly in your angular.json
+The Component Extractor is accessible via this NgCLI builder: `@o3r/components:extractor`.
 
-First thing to do is to define your given filenames for the configuration in the _package.json_ of the library/app where you run the extractor.
-When running in a library it will use this configuration as the names for the metadata files.
-When running the extractor in an application, it will search for these filenames in each node_module (package.json file) of each library,
-in order to concat the metadata from the file with other libraries metadata and app metadata.
+First, define the file names of the package metadata in the `package.json` of the library/application where you run the extractor.
+When the extractor is run in a project, it will use these file names to name the output files of the extraction. In case you want to compile the
+metadata of the project with its dependencies (generally for an application that has libraries with their own metadata as well), the extractor will need to find the metadata for each dependency.
+In that case, the extractor will search for these file names in the `package.json` file of each library (in the `node_modules`) 
+in order to concatenate the file's metadata with the metadata of other libraries and the application's metadata.
 
+In the `package.json` of the library:
+```json
+"cmsMetadata": {
+  "componentFilePath": "./component.class.metadata.json",
+  "configurationFilePath": "./component.config.metadata.json"
+}
 ```
-// in package.json file
-...
-  "cmsMetadata": {
-    ...
-    "componentFilePath": "./component.class.metadata.json",
-    "configurationFilePath": "./component.config.metadata.json",
-  }
-...
-```
-
-The Component Extractor is accessible via a NgCLI builder: `@o3r/components:extractor`;
 
 For an up-to-date documentation, run `ng help @o3r/components:extractor`
 
-* If the _component extractor_ is run on an application which is using components from an otter library, the _libraries_ option can be specified to concat classes/configuration files generated for application with ones from specified _libraries_. The extractor will search a _'component.config.metadata.json'_ and a _component.class.metadata.json_ file in the node_modules package of each defined library (the name of the files to search will be read from the _cmsMetadata_ property in _package.json_ file defined for each library).
-* Here is an example on the otter demo app
-
-```
-// in angular.json
+* If the __component extractor__ is run on an application with components from an Otter library, the `libraries` option can be used to concatenate the metadata files generated for the application with the ones from the specified libraries.
+  The extractor will search for the component configuration and style metadata files in the `node_modules` package of each configured library.
```

Comment:
Do not explicitly speak about `node_modules`, we should prefer `dependencies` to indicate the support of non-NPM clients